### ISSUE TITLE
Demo where steps can be set via URL hash

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#91]
So it looks in `window.location.hash` to find a comma delimited set of steps, like `#steps=ndvi,threshold`, and adds them to the reconfigurable demo. 

As you remove/add steps, we update the URL hash to reflect this. 

That way, we can pass around almost any imaginable demo sequencer as a fixed URL -- very useful!

This would come after #82 (completed!). Some example of reading/writing the URL hash can be found at https://github.com/jywarren/fisheyegl/blob/master/example/urlHash.js

**Comment by jywarren** [comment]
So, for example, if someone says "I wish there were a fast way to crop and threshold these images, you could send them a link to the demo with the right steps configured, and they'd go and it'd be all set up for them, with no having to change the demo code :-)

**Comment by ccpandhare** [comment]
Yes, I agree to this. I shall do that. Now working on #82 :-)

**Comment by jywarren** [comment]
awesome!

**Comment by jywarren** [comment]
This will be great. I think we can even embed a customized sequencer into a wiki page or research note on PublicLab.org -- it'll be a great way to put the library into use!

**Comment by ccpandhare** [comment]
Hope to do that soon :-)
Will get the demo done by tomorrow, and then we can work on this (#91) and then the wiki page!
This will be good documentation too.

**Comment by ccpandhare** [comment]
The demo is pretty much done for now, so I shall start working on this

**Comment by jywarren** [comment]
Great. Sorry for being offline much of this week, it was a vacation I really needed! I'll be back online this coming week.

**Comment by ccpandhare** [comment]
What if the user wants to set any options? I think that should be included in the hash as well.

**Comment by jywarren** [comment]
Sure, but then the hash may get pretty long. Let's see how it does though.
Maybe we can either offer to save the options with a checkbox or we can
just clearly label options so they can be cut out of the URL if people
want. But yes let's start by including them. Good idea!

**Comment by ccpandhare** [comment]
This is pretty clear.
I am just skeptical about adding options to steps via the hash, and if we don't, the incompleteness of the same.

Yes, we should start including them.

**Comment by jywarren** [comment]
What do you think about a syntax more like: 

`examples/#steps=ndvi,threshold[40]` -- although this starts to get a bit weird, and we'd have to specify the keyname too, so `[value=40]` ... just brainstorming here.

**Comment by jywarren** [comment]
also I added you on npmjs.org - you should be able to `npm publish` now, but please take care to do so only from the master branch.

**Comment by jywarren** [comment]
Now that I think of it, let's just break this in two. First version should have just steps, no options. It'll just use the defaults. Let's open another issue (which we can think about a bit longer) that has options presets too. That more extended version shouldn't stop us from doing the initial version first.

**Comment by ccpandhare** [comment]
Looks cool. Will do!

**Comment by jywarren** [comment]
Completed and split off new issue!
